configure.ac

```diff
@@ -1090,24 +1090,6 @@ AC_MSG_NOTICE([checking required package dependencies])
 # NL_WITH_PACKAGE(...)
 
 #
-# Checks for missing functions
-#
-
-AC_DEFUN([CHECK_MISSING_FUNC], [
-    AC_CHECK_FUNC($1, [], [
```

Comment:
I think we should keep the feature tests for the missing functions in the configure script, defining `HAVE_*` macros for each missing function. Individual platforms (Like windows) can override these if necessary.

So this would be simplified to just being something like...

```
AC_CHECK_FUNCS([strlcpy strlcat strncpy])
```